### DISCUSSION TITLE
"Clear" Button for WEB Console ?

### DISCUSSION THREAD

**Discussion by Bionic-Bean** [arendst/Tasmota#13299] [Ideas]
Hi Guys,
I appreciate memory space is at a premium, but would it be nice to have a "Clear" console window button on the web console screen. When debugging and logging new data flowing into the console it would be nice to clear the window from time to time to empty to allow new data to be seen easily rather than having to wade though the data already present in the consol. Not a major issue, I know but a nice feature to have and guessing would not add much code to implement. !! 
IS this doo'able ?  Would anybody else like a 'Clear' button on the web console screen?
Have Fun Dave

**Comment by arendst** [comment]
Just do a refresh of the screen to clear the console and only see the latest logged messages.

Simple and at no code costs.

**Reply by sfromis** [reply]
What I've done a few times was to type some command (like `Time`) to have a visual break when I'm expecting similar lines to appear.

**Reply by belowm** [reply]
I guess that is not (no longer?) valid, since recent log messages are kept. Maybe implement a command to clear the console?
